package.json

```diff
@@ -61,7 +61,7 @@
     "redux": "^4.0.1",
     "redux-logger": "^3.0.6",
     "redux-thunk": "^2.3.0",
-    "sequelize": "^5.3.1",
```

Comment:
same (Have you changed these dependencies for some reason?)

Reply:
My changes/ differences can be deleted